### PR TITLE
[Chore #150397534] setup unit testing for frontend

### PR DESCRIPTION
**Pull request by ayeniblessing101**
#### What does this PR do?
Setup unit testing for the frontend

#### Description of Task to be completed?
- Installed karma, jasmine-core, karma-jasmine,
 karma-chrome-launcher

#### How should this be manually tested?
- Run karma start

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
setup unit testing for the frontend #150397534

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A

### CODE REVIEW

File: test/index.spec.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+describe('Home factory', function() {
+  it('has a dummy spec to 2 + 2', function(){
+    expect(2 + 2).toEqual(4);
+  })
```
[the remark this answers is not in the document]
Comment (THEozmic) [reply test/index.spec.js]: Please fix this. Thanks

File: karma.conf.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
+// Karma configuration
+// Generated on Tue Aug 22 2017 12:53:11 GMT+0100 (WAT)
+
+module.exports = function(config) {
+  config.set({
+
+    // base path that will be used to resolve all patterns (eg. files, exclude)
+    basePath: '',
+
```
Comment (mradeybee) [inline karma.conf.js:9]: Remove extra space here.

File: karma.conf.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+    // frameworks to use
+    // available frameworks: https://npmjs.org/browse/keyword/karma-adapter
+    frameworks: ['jasmine'],
+
+
```
Comment (mradeybee) [inline karma.conf.js]: Ditto extra space.

File: karma.conf.js [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+    // Continuous Integration mode
+    // if true, Karma captures browsers, runs the tests and exits
+    singleRun: false,
+
+    // Concurrency level
+    // how many browser should be started simultaneous
+    concurrency: Infinity
+  })
```
[the remark this answers is not in the document]
Comment (mradeybee) [reply karma.conf.js]: Always listen to hound CI!

**Review by mradeybee — CHANGES_REQUESTED** [review]
@ayeniblessing101 This looks okay. I will suggest you add a template of what the test should look like so it will be easier for the next person to write a test to do so. 

Also, in your `karma.config.js` file, make sure the space between the lines are reduced to one. You can ignore if it is an auto-generated file.

**Review by mradeybee — APPROVED** [review]
GTM

**Pull request merged**